src/setupTests.ts

```diff
@@ -81,3 +99,8 @@ global.jestSpyConsole = async (method: ConsoleMethod, fn: JestSpyConsoleFn, opti
   // using the mock after restore will still generate an error at runtime
   return undefined as any as typeof consoleMethodSpy;
 };
+
+global.assertIsDefined = (value: unknown) => {
+  expect(value).toBeDefined();
+  expect(value).not.toBeNull();
+};
```

Comment:
Maybe we should publish these as a `@concord-consortium/jest-utils` npm module. I've thought about the fact that `jestSpyConsole` could be useful in other projects and the same is true for `assertIsDefined`.